packages/guides/src/Nodes/InlineToken/GenericTextRoleToken.php

```diff
@@ -4,7 +4,7 @@
 
 namespace phpDocumentor\Guides\Nodes\InlineToken;
 
-class GenericTextRoleToken extends ValueToken
+class GenericTextRoleToken extends InlineMarkupToken
```

Comment:
Suggestion: drop the name Token as these are nodes now?

Reply:
Done